### ISSUE TITLE
Cant run needrestart - Cant locate Module/Find.pm

### ISSUE THREAD

**Issue by og900aero** [liske/needrestart#312]
I'm on Debian 12.7. When I run needrestart, or needrestart -v, needrestart --help, I get error message:

_Can't locate Module/Find.pm in @INC (you may need to install the Module::Find module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.36.0 /usr/local/share/perl/5.36.0 /usr/lib/x86_64-linux-gnu/perl5/5.36 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl-base /usr/lib/x86_64-linux-gnu/perl/5.36 /usr/share/perl/5.36 /usr/local/lib/site_perl) at /usr/share/perl5/NeedRestart.pm line 29.
BEGIN failed--compilation aborted at /usr/share/perl5/NeedRestart.pm line 29.
Compilation failed in require at /usr/sbin/needrestart line 31.
BEGIN failed--compilation aborted at /usr/sbin/needrestart line 31._

**Comment by liske** [comment]
This sounds like a problem in the packaging in Debian. Please open a bug at the [Debian BTS](https://www.debian.org/Bugs/Reporting) if you are using their official packages.

**Comment by og900aero** [comment]
> This sounds like a problem in the packaging in Debian. Please open a bug at the [Debian BTS](https://www.debian.org/Bugs/Reporting) if you are using their official packages.

No, I don't use repo version, I did the following:

1. git clone https://github.com/liske/needrestart.git
2. cd needrestart
3. sudo make install

**Comment by liske** [comment]
then you have to install the [dependencies](https://github.com/liske/needrestart/blob/master/INSTALL.md) yourself

**Comment by og900aero** [comment]
Only the po-debconf package is missing. I installed it now, but the error message remained. The Find.pm module is missing (from Perl?). But I don't know how to install it.

**Comment by liske** [comment]
https://packages.debian.org/file:Module/Find.pm

**Comment by og900aero** [comment]
> https://packages.debian.org/file:Module/Find.pm

Thx. I tried it, and it turned out that a couple of such modules were still missing, which I now added in this way, so it works well now.
It is interesting that at the beginning when you did the repair so that it would work fine with the xanmod kernel, then I did not have such a problem. I suspect that apt autoremove detected that these modules are not needed, since the needrestart in the repo is not installed, so it removed it. After that, I will monitor what the apt program will indicate. Thank you for your help!
